src/main/groovy/life/qbic/datamodel/dtos/business/facilities/Facility.groovy

```diff
@@ -17,22 +17,35 @@ package life.qbic.datamodel.dtos.business.facilities
  */
 enum Facility {
 
-    CFMB("Core Facility for Medical Bioanalytics"),
-    IMGAG("Institute for Medical Genetics and Applied Genomics"),
-    MGM("Institute for Medical Microbiology and Hygiene"),
-    QBIC("Quantitative Biology Center"),
-    PCT("Proteome Center Tübingen")
+    CFMB("Core Facility for Medical Bioanalytics", "CFMB"),
+    IMGAG("Institute for Medical Genetics and Applied Genomics", "IMGAG"),
+    MGM("Institute for Medical Microbiology and Hygiene", "MGM"),
+    QBIC("Quantitative Biology Center", "QBIC"),
+    CFMB_PCT("Proteomics Facility Tübingen", "Proteomics Facility")
 
     private final String fullName
+    private final String label
 
     /**
      * Creates an instance of a facility enum
-     * @param fullName The full name representation of the enum
+     * @param fullName The full name representation of the facility
      */
     Facility(String fullName) {
         this.fullName = fullName
+        this.label = ""
     }
```

Comment:
Why do you provide this constructor. In case a label is always expected (seems like it from the current emum objects) this constructor will not be used and can be removed.